### DISCUSSION TITLE
Need help to use Vercel Nameservers

### DISCUSSION THREAD

**Discussion by leo-mathurin** [vercel/community#3135] [Help]
### Summary

I was trying to add DNS Records to my Vercel Project, but they don’t seem to be working, and being a complete noobie in DNS´s, I can’t find a solution with the Vercel documentation. As you can see in the picture, I did add the different records required for my Elastic Mail form to work, but when I check on Elastic Mail if they are working, they’re completely not, even though I waited more than enough for them to be applied (literally two weeks). So I guessed that it would be because I didn’t use Vercel Nameservers as told in the small yellow warning. But I cannot find a way to use them :(
My project is vanilla HTML/CSS/JS, and i kept every option on default when I was deploying it. 
I would be very grateful if someone would take the time to help me.
![image](https://github.com/vercel/community/assets/114407386/3db6ef92-a822-40f5-9a61-3f8d4ff5cf77)

### Steps to Reproduce

-Deploy a default HTML/CSS/JS project with an Elastic Mail e-mail form
-Add the corresponding DNS Records

**Comment by amyegan** [comment]
Thanks for reaching out, @lelemathrin. DNS configuration can definitely feel overwhelming if you aren't used to it!

It looks like your domain is pointed to your Vercel project with `A` and `CNAME` records from outside of Vercel. They are most likely at the registrar where you originally bought the domain (something like Google Domains, Namecheap, GoDaddy, etc). To set that up, you would have gone through a process like this:  https://vercel.com/docs/concepts/projects/domains/add-a-domain

If your domain is connected to your project via `A` and `CNAME` records, that means the elasticmail records should be added in the same place that those records were created.

**Reply by amyegan** [reply]
If you prefer to use Vercel nameservers instead of updating the other records, you could do that by changing some settings with your domain registrar. You can learn more about that in our documentation on [working with nameservers](https://vercel.com/docs/concepts/projects/domains/working-with-nameservers) and in the documentation provided by your domain name registrar.

**Reply by leo-mathurin** [reply]
Well, the thing is that I didn't get any domain out of Vercel, I just added a free custom one with the "Domains" page of my project when I deployed, all just with Vercel (it finishes with ".vercel.app"). So I was wondering why would it be Third Party if I got it for free on Vercel. I linked the Domain of my project and the one I'm supposed to have got from Third Party, maybe they're not the same one ?
![image](https://github.com/vercel/community/assets/114407386/911b988e-df59-403c-8ded-b675d3f6f212)
![image](https://github.com/vercel/community/assets/114407386/dda90f13-5252-45d0-a5cd-6d2eddd45d84)
